t/perf/p7519-fsmonitor.sh

```diff
@@ -136,7 +136,11 @@ test_expect_success "one time repo setup" '
 
 setup_for_fsmonitor() {
 	# set INTEGRATION_SCRIPT depending on the environment
-	if test -n "$INTEGRATION_PATH"
+	if test -n "$USE_FSMONITOR_DAEMON"
+	then
+		git config feature.fsmonitor true &&
+		INTEGRATION_SCRIPT=false
+	elif test -n "$INTEGRATION_PATH"
 	then
 		INTEGRATION_SCRIPT="$INTEGRATION_PATH"
```

Comment:
We should probably unset `feature.fsmonitor` for symmetry here.

Reply:
Done.